client/src/components/Tag/Tag.js

```diff
@@ -61,7 +61,7 @@ const Tag = ({
       role="button"
       className={classnames('tag', 'btn-sm', { tag__deletable: deletable })}
       onClick={(e) => { e.preventDefault(); onClick(dataKey); }}
-      tabIndex={focusable ? undefined : -1}
+      tabIndex={focusable ? -1 : undefined}
```

Comment:
You sure about this? Setting the focus to -1 prevents it for getting focus.

Reply:
Yea, they used to be `button`s but are now `li`s which aren't focusable by `.focus()`, adding `-1` allows them to be targeted by `.focus()`. We handle the focus programmatically anyway rather than relying on the default tab order. More info: https://snook.ca/archives/accessibility_and_usability/elements_focusable_with_tabindex

Reply:
But people won't be able to access the tags with the tab key. I'm not against adding extra ways to navigate the CMS, but I'm kind of concern with subverting standard ways.

If you're using a screen reader, this will be very confusing.

Reply:
Ah, it was a regression before this change because we changed from `button` to `li` and we needed to make `li` focusable not just through backspace/js – I've changed it to 0 which allows keyboard focus